spec/javascripts/fixtures/EmbeddedGoogleAnalytics.html

```diff
@@ -0,0 +1,34 @@
+<div>
+  <div class="analytics-charts" style="display:none;" data-dough-component="EmbeddedGoogleAnalytics" data-dough-embedded-google-analytics-config='{"path": "/en/articles/foo"}'>
+    <div class="l-constrained">
+      <h2>Google Analytics</h2>
+      <div id="embed-api-auth-container"></div>
+
+      <div class="l-2col">
```

Comment:
Don't think we need these wrapping divs in the fixture, probably worth removing to make them more generic.